styleguide/_themes/derek/scss/patterns/grid/businessResource.scss

```diff
@@ -1,14 +1,24 @@
-.resource {
+$hover-color: 12px solid rgba(0, 0, 0, .4);
```

Comment:
That came in via Sam's changes in PR #211. I didn't catch it in the code review, but that's probably unnecessary so I'll just pull it out here.